blobit-services/src/test/java/blobit/server/SimpleClusterTest.java

```diff
@@ -57,12 +57,17 @@ public void test() throws Exception {
 
                 props.put(ServerConfiguration.PROPERTY_BASEDIR, folder.newFolder().getAbsolutePath());
                 props.put(ServerConfiguration.PROPERTY_ZOOKEEPER_ADDRESS, zookeeperServer.getConnectString());
+                props.put(ServerConfiguration.PROPERTY_BOOKKEEPER_ZK_LEDGERS_ROOT_PATH, "/custom-path");
```

Comment:
For your joy @diegosalvi the test is testing that it is really possible to use a custom path